source/includes/reference/_comparison.md

```diff
@@ -0,0 +1,208 @@
+includes not only TransferWise but other providers in the market.
+
+
+### Price Estimation
+
+The quotes (price / speed) provided by this API are based off of real quotes collected from 3rd party websites. We collect both
+the advertised exchange rate and fee for each provider for various amounts. When a comparison is requested we calculate the markup %
+on the collected exchange rate on the mid-market rate at the time of collection, we then apply this markup % on the current 
+mid-market rate to provide a realistic estimate of what each provider offers. We collect data for all providers ~ once per hour to 
+ensure we provide as accurate and up to date information as possible.
+
+Note: Today, we only provide estimations for FX transactions with a Bank Transfer pay-in / pay-out option. This is important to stress
+as many providers offer significantly different fees / exchange rates when used debit / credit card, cash etc.
+
+For more details on the data collection process please see the following page:
+
+[https://transferwise.com/gb/compare/disclaimer](https://transferwise.com/gb/compare/disclaimer)
+
+If you have questions or suspect any data to be inaccurate or incomplete please contact us at:
+
+comparison@transferwise.com
+
+
+### Delivery Estimation
+
+Similar to price, we collect speed data for most (if not all) providers which we have price information for. Many providers
+display speed estimates to their customers in a number of different ways. 
+
+Some examples:
+
+- "The transfer should be complete within 2-5 days"
+- "The money should arrive in your account within 48 hours"
+- "Should arrive by 26th Aug"
+- "Could take up to 4 working days"
+
+The below API intends to model these in a consistent format by providing a min / max range for all delivery estimations.
+An estimate that states "up to X" will have "max" set to a duration but "min" as null, a "from X" will have "min" set to a
+duration and "max" as null. Finally for those providers who offer a specific, point in time estimation (like TransferWise), 
+the API will surface a duration where min/max are equal. 
+
+
+### Quotes structure
+
+In order to provide the most flexible and accurate data for clients, we surface a denormalised list of quotes per provider 
+where each quote represents a unique collection of comparison "dimensions". 
+
+For example, for a given request for all comparisons from GBP to EUR for £1000 i.e:
+
+```shell
+curl -X GET https://api.transferwise.com/v3/comparisons/?sourceCurrency=GBP&targetCurrency=EUR&sendAmount=1000
+```
+
+A single given provider may expose multiple quotes for the same currency route. The most common example is where a provider 
+offers different pricing for one country vs another country which uses the same currency. e.g:
+
+Provider X:
+
+- GBP EUR 1000 [GB -> ES] fee: 10, rate: 1.5
+- GBP EUR 1000 [GB -> ES] fee: 8, rate: 1.5
+- GBP EUR 1000 [GB -> ES] fee: 10, rate: 1.35
+
```

Comment:
Is this supposed to be different countries and not just ES?

Reply:
yes... thank you!